### DISCUSSION TITLE
Documentation for next-auth usage with NextJS 13 seems unclear / non-existent

### DISCUSSION THREAD

**Discussion by sarangzambare** [nextauthjs/next-auth#7946] [Help]
I want to implement nextauth in my nextjs 13 app, but most of the documentation assumes the **pages/** folder and the **_app.tsx file** - which do not exist in nextjs 13 + 

there are many tutorials out there that teach you how to use nextauth in nextjs 13, but i could not find how to do it in the official docs. 

Am I missing something?

**Comment by scottishsmile** [comment]
It is an absolute sh** show mate. Just wait until you try to put it into production behind a proxy. Everything was good in Dev but now 8 things are broken and there's no guide if it's not deployed on vercel. They also don't do refresh tokens, you have to code that yourself and it won't work, so it's really a hobby app from the 2000s. Just use Auth0 or Google firebase.

**Reply by sarangzambare** [reply]
damn.. i was very close to ripping my hair out with this shit, thanks for clarifying that its only gonna get worse in prod. curious what did you end up using?

**Reply by pschaub** [reply]
@scottishsmile Could you let us know your exact issues you have in production? Maybe this could be solved soon. If possible it would be helpful to have an example on GitHub to reproduce the issue.

**Comment by sanchitwadehra** [comment]
It's 2025 and I am here just to confirm that it is still shit literally everything in documentation doesn't work the way it's mentioned please stay away from this
